commons/src/main/java/org/infinispan/counter/api/CounterConfiguration.java

```diff
@@ -213,7 +213,7 @@ public CounterConfiguration build() {
       }
    }
 
-   private static class Externalize implements AdvancedExternalizer<CounterConfiguration> {
+   public static class Externalize implements AdvancedExternalizer<CounterConfiguration> {
```

Comment:
why does it need to be public?

Reply:
I don't think they do ... this is a leftover from a WIP branch I had.

Reply:
Also we are missing an `r` on the end of the name there ;)